public/app/plugins/datasource/cloud-monitoring/plugin.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "Stackdriver",
+  "name": "Cloud Monitoring",
```

Comment:
I've read Cloud Monitoring in several places and in some places Google Cloud Monitoring. Think we should try to be consistent.